src/Hedgehog/Range.fs

```diff
@@ -8,9 +8,10 @@ open Hedgehog.Numeric
 type Size = int
 
 /// A range describes the bounds of a number to generate, which may or may not
-/// be dependent on a 'Size'.
+/// be dependent on a 'Size'.  The pair returned by the function contains the
+/// lower bound on the left and the upper bound on the right.
```

Comment:
We can also mention the origin `'a` which is the origin between the lower and upper bound, and that 
`(Size -> 'a * 'a)` is a function from 'Size' to bounds.  As the size goes towards 0, the values go towards the origin.

Reply:
I added a sentence in a commit that I just force pushed.  Let me know if you would prefer to say more.

Reply:
It's good enough. In another PR you may also (freely) copy some of the docs from Range.hs (Haskell) since this is a port from there.